internal/services/appservice/windows_web_app_resource_test.go

```diff
@@ -4198,3 +4213,33 @@ resource "azurerm_windows_web_app" "test" {
 }
 `, r.baseTemplate(data), data.RandomInteger, data.RandomInteger)
 }
+
+func (r WindowsWebAppResource) alwaysOnFalse(data acceptance.TestData) string {
+	return fmt.Sprintf(`
+provider "azurerm" {
+  features {}
+}
+
+%s
+
+resource "azurerm_windows_web_app" "test" {
+  name                = "acctestWA-%d"
+  location            = azurerm_resource_group.test.location
+  resource_group_name = azurerm_resource_group.test.name
+  service_plan_id     = azurerm_service_plan.test.id
+
+  site_config {
+    always_on           = true
```

Comment:
I believe this should be `false`?

```suggestion
    always_on           = false
```

Reply:
Taken care of. ty ty